app/views/api/v1/informations/_info.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.extract! information, :text
```

Comment:
You should be consistent. You're not always using extract!. Use extract! always or not, but try to use the same approach in all the project.